cadquery/FCAD_script_generator/GW_QFP_SOIC_SSOP_TSSOP_SOT/cq_parameters_tssop.py

```diff
@@ -0,0 +1,1545 @@
+#        L = 0.6,         # pin bottom flat part length (including corner arc)
+        fp_s = 1,          # True for circular pinmark, False for square pinmark (useful for diodes)
+        fp_r = 0.4,          # First pin indicator radius
+        fp_d = 0.5,          # First pin indicator distance from edge
+        fp_z = 0.05,       # first pin indicator depth
+        ef = 0.0,          # fillet of edges  Note: bigger bytes model with fillet
+        cc1 = 0.25,        # 0.45 chamfer of the 1st pin corner
+        D1 = 9.7,         # body length
+        E1 = 6.1,         # body width
+        E = 8.1,          # body overall width
+        A1 = 0.2,          # body-board separation
+        A2 = 1.0,          # body height
+        b = 0.26,          # pin width
+        e = 0.65,          # pin (center-to-center) distance
+        npx = 15,           # number of pins along X axis (width)
+        npy = 0,           # number of pins along y axis (length)
+        epad = None,       # e Pad
+        excluded_pins = None,          # pin excluded
+        old_modelName = 'TSSOP-30_6.1x9.7mm_P0.65mm',            # modelName
+        modelName = 'TSSOP-30_6.1x9.7mm_P0.65mm',            # modelName
+        rotation = -90,      # rotation if required
+#        dest_dir_prefix = '../Package_SO.3dshapes',      # destination directory
+        ),
+
+    'TSSOP-44_4.4x11mm_P0.5mm': Params(
+        #
+        # TSSOP, 44 Pin (JEDEC MO-153 Var BE https://www.jedec.org/document_search?search_api_views_fulltext=MO-153), generated with kicad-footprint-generator ipc_gullwing_generator.py
+        # This model have been auto generated based on the foot print file
+        # A number of paramters have been fixed or guessed, such as A2
+        #
+        # The foot print that uses this 3D model is TSSOP-44_4.4x11mm_P0.5mm.kicad_mod
+        #
+        the = 9.0,         # body angle in degrees
+        tb_s = 0.1,       # top part of body is that much smaller
+        c = 0.1,           # pin thickness, body center part height
+        R1 = 0.1,          # pin upper corner, inner radius
+        R2 = 0.1,          # pin lower corner, inner radius
+        S  = 0.1,          # pin top flat part length (excluding corner arc)
+#        L = 0.6,         # pin bottom flat part length (including corner arc)
+        fp_s = 1,          # True for circular pinmark, False for square pinmark (useful for diodes)
+        fp_r = 0.4,          # First pin indicator radius
+        fp_d = 0.5,          # First pin indicator distance from edge
+        fp_z = 0.05,       # first pin indicator depth
+        ef = 0.0,          # fillet of edges  Note: bigger bytes model with fillet
+        cc1 = 0.25,        # 0.45 chamfer of the 1st pin corner
+        D1 = 11.0,         # body length
+        E1 = 4.4,         # body width
+        E = 6.4,          # body overall width
+        A1 = 0.1,          # body-board separation
+        A2 = 1.5,          # body height
```

Comment:
Why? Max overall height for all variations is 1.2mm

Reply:
Thanks for pointing it out. Corrected in #339